scd30/scd30.c

```diff
@@ -80,9 +80,8 @@ int16_t scd30_read_measurement(float *co2_ppm, float *temperature,
     int16_t ret;
     union {
         uint32_t u32_value;
-        float float32;
         uint8_t bytes[4];
-    } tmp, data[3];
+    } data[3];
```

Comment:
We can get rid of the whole `union` here and just use a byte array, right?

Reply:
..although readability is not necessarily better:

```
uint8_t bytes[3 * 4];
// ...
*temperature = sensirion_bytes_to_float(bytes + 4);
*humidity = sensirion_bytes_to_float(bytes + 8);
```

Reply:
What about `uint8_t data[3][4]`? Then you can use 
```c
*co2_ppm = sensirion_bytes_to_float(data[0]);
*temperature = sensirion_bytes_to_float(data[1]);
*humidity = sensirion_bytes_to_float(data[2]);
```